packages/lib-runtime/src/utils/objects.ts

```diff
@@ -17,7 +17,9 @@ export const createMethodDelegate = <
 ) => {
   const delegate = {} as Pick<TObject, TObjectMethods>;
 
-  const validMethodNames = methodNames.filter((methodName) => _.isFunction(target[methodName]));
+  const validMethodNames = methodNames.filter(
+    (methodName) => typeof target[methodName] === 'function',
```

Comment:
Using `typeof obj === 'function'` seems generally better than `_.isFunction` since the former is natively supported by the browser.

Please correct me if I'm wrong here :smiley:

Reply:
I honesty think Lodash is overrated for a lot of the basic functions of JS. Deep copies is the only thing I think of from a use-case of Lodash. Most of their functions are minutely better or not at all. I benched a few when I first got here because my previous team used lodash for absolutely everything and it was just making it harder to read.

So yes, native 👍